hydra/core/override_parser/types.py

```diff
@@ -256,10 +275,13 @@ def value(
         else:
             return Override._convert_value(self._value)
 
-    def sweep_string_iterator(self) -> Iterator[str]:
+    def sweep_iterator(
+        self, transformer: TransformerType = Transformer.identity
+    ) -> Iterator[ElementType]:
         """
-        Converts the sweep_choices from a List[ParsedElements] to a List[str] that can be used in the
-        value component of overrides (the part after the =)
+        Converts CHOICE_SWEEP, SIMPLE_CHOICE_SWEEP, GLOB_CHOICE_SWEEP and RANGE_SWEEP to a List[Elements].
+        The returned sweep.
+        A transformer may be provided for converting each element to support the needs of different sweepers
```

Comment:
The comment still does not read right.

Reply:
I didnt even type them myself - I just used the "commit" option fro github. Will check this :)

Reply:
Yes. I typed them. it's a bit hard to edit things in the GitHub UI.

Reply:
ahh my bad :) Fixing it